report/2.Analysis/EducationalMiniComputers.tex

```diff
@@ -0,0 +1,30 @@
+\section{Educational mini computers}
+
+Apart from the Arduino board, a lot of other mini computers have been created with educational purposes in mind.
+To figure out why these "mini computers" are so popular for technical learning, some of the most popular ones were analysed.
+This should help to figure out which functionality the HCL programming language should include in order to comply with the intend of the mini computers.
+
+\subsection{Raspberry Pi}
+The Raspberry Pi is one of the most widely known single board computers. 
+It is a complete arm-based computer, and may run anything arm based on it, including both linux and windows based arm operating systems. 
+This first of all makes it a common entry point into the GNU/Linux operating systsem stack, which is generally considered to be very educational in regards to how computers work.
```

Comment:
* Spelling mistake. "systsem" should be "system"
* "Very" used as an adverb is implying some undifined measument of great size and therefore not optimal when used to describe something that is non-trivial to measure like begin "educational".

Reply:
Fixed spelling mistake. But very is still present. I don't feel like it's too big of an issue, but if you have a great alternative sentence let me know, and I will fix it.